Makefile

```diff
@@ -378,8 +378,8 @@ package-capi:
 	cp lib/c-api/wasm.h* package/include
 	cp lib/c-api/README.md package/include/README.md
 ifeq ($(OS), Windows_NT)
-	cp target/release/wasmer_c_api.dll package/lib
-	cp target/release/wasmer_c_api.lib package/lib
+	cp target/release/wasmer_c_api.dll package/lib/wasmer.dll
+	cp target/release/wasmer_c_api.lib package/lib/wasmer.lib
```

Comment:
It'd be good to check with Syrus on the name inconsistency, if I remember correctly they've always had a different name than other platforms and I believe I've verified that that was intentional in the past

Reply:
I just re-checked. This looks good!